hazelcast/src/main/java/com/hazelcast/internal/util/executor/CompletableFutureTask.java

```diff
@@ -57,16 +58,9 @@ public void run() {
         }
 
         try {
-            Callable c = callable;
-            if (c != null) {
-                Object result = null;
-                try {
-                    result = c.call();
-                    complete((V) result);
-                } catch (Throwable ex) {
-                    completeExceptionally(ex);
-                }
-            }
+            complete(callable.call());
```

Comment:
What do you think about preserving original code? I am sure, it's easier to do a null-check here, than guarantee null-safety for all callers.